addons/api/mirage/generated/factories/target.js

```diff
@@ -9,6 +9,7 @@ export default Factory.extend({
   name: () => random.words(),
   session_max_seconds: () => datatype.number(),
   session_connection_limit: () => datatype.number(),
+  worker_filter: () => random.words(),
```

Comment:
What's the difference between the usage of `addons/api/mirage/generated/factories/target.js` vs `addons/api/mirage/factories/target.js`? It seems like the former overrides/extends the latter?

Reply:
In a perfect world, we would automatically generate models and mocks from the open API schema definition.  The code is structured in a way to allow this, though no such automation yet exists.  This arrangement allows us to override the generated code if we need to _without directly editing the generated files_.